packages/core-bridge/ts/native.ts

```diff
@@ -175,6 +185,7 @@ export interface WorkerOptions {
   identity: string;
   buildId: string;
   useVersioning: boolean;
```

Comment:
Continuing on my previous comment, remember that anything in native.ts are 100% internal APIs, so there is absolutely no reason to preserve fields that are no longer needed, or to preserve a legacy structure that no longer reflect the way they are consumed in Core. The presence of `buildId` and `useVersioning` directly in `WorkerOptions` are examples of such legacy things that we'll want to avoid from now on.

Again, I'm just explaining the direction I want to encourage, but don't feel obliged to fix in this PR. I can take care of this later.

Reply:
They're still kinda needed as they're how you select the old versioning strategy. We could pass _that_ straight through to the bridge though, but, I figure we can just remove them once they're no longer available at all.